ci/functions.sh

```diff
@@ -127,7 +127,7 @@ function run_smoke_tests_stage() {
     echo "Running E2E tests with IQE:"
     echo "IQE_MARKER_EXPRESSION: '$IQE_MARKER_EXPRESSION'"
     echo "IQE_FILTER_EXPRESSION: '$IQE_FILTER_EXPRESSION'"
-
+    export IQE_IMAGE_TAG="cost-management-1891-e26f8349"
```

Comment:
Curious why we are adding this `IQE_IMAGE_TAG`?

Reply:
@djnakabaale this is a temporary solution to run this PR against a modified iqe plugin version :)

Reply:
@bacciotti would you be interested in documenting how you created the tag and how you can use this env var to run  the smokes off a modified version? I think it could be helpful for the team moving forward.

Reply:
FTI I've created a document explaining this. Please ping me on slack, then I can send you.